shumai/tensor/tensor.ts

```diff
@@ -457,15 +472,7 @@ export class Tensor {
   }
 
   get shape64() {
```

Comment:
this change isn't quite right

Reply:
Oh my bad, auto lint action on save might have caused this change haha